kuma-2.8.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: kuma-2.8
+  version: 2.8.2
+  epoch: 0
+  description: A multi-zone service mesh for containers, Kubernetes and VMs.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kumahq/kuma
+      tag: ${{package.version}}
+      expected-commit: ffb0a135b8320cd474c503b785b43627d77a1338
+
+subpackages:
+  - name: kuma-cp-2.8
```

Comment:
I'm not sure but it should be kuma-2.8-cp because its a sub-package of kuma-2.8 package, let's use ${{package.name}}-cp something like that.

Reply:
I have this named as kuma-cp-2.x in enterprise ATM
https://github.com/chainguard-dev/enterprise-packages/pull/6060

Reply:
I'm not sure what the standard is here but I mostly have used ${{package.name}}-<subpackage_name> pattern, @EyeCantCU any thoughts on this?

Reply:
I think we should do something like this:

```bash
var-transforms:
  - from: ${{package.version}}
    match: ^(\d+)\.\d+$
    replace: "$1"
    to: major-minor-version
```

And call it `kuma-cp-${{vars.major-minor-version}}` now that subpackage names support vars